app/register-organization/page.tsx

```diff
@@ -0,0 +1,283 @@
+                            `}
+                        />
+                      </Button>
+                    </PopoverTrigger>
+                    <PopoverContent className="p-0">
+                      <Command>
+                        <CommandInput placeholder="Search country..." />
+                        <CommandEmpty>No country found.</CommandEmpty>
+                        <CommandGroup>
+                          <CommandList>
+                            {COUNTRIES?.map((country) => (
+                              <CommandItem
+                                key={country?.value}
+                                value={country?.value}
+                                onSelect={() => {
+                                  form.setValue("country", country?.value);
+                                  setCountryPopover(false);
+                                }}
+                              >
+                                <Check
+                                  className={cn(
+                                    "mr-2 h-4 w-4",
+                                    field?.value === country?.value
+                                      ? "opacity-100"
+                                      : "opacity-0",
+                                  )}
+                                />
+                                {country?.label}
+                              </CommandItem>
+                            ))}
+                          </CommandList>
+                        </CommandGroup>
+                      </Command>
+                    </PopoverContent>
+                  </Popover>
+
+                  <FormMessage />
+                </FormItem>
+              )}
+            />
+
+            <FormField
+              control={form.control}
+              name="timeZone"
+              render={({ field }) => (
+                <FormItem className="flex flex-1 flex-col">
+                  <FormLabel required>TimeZone</FormLabel>
+
+                  <Popover
+                    open={timeZonePopover}
+                    onOpenChange={setTimeZonePopover}
+                  >
+                    <PopoverTrigger asChild>
+                      <Button
+                        variant="popover"
+                        className={`${field?.value === "" ? "text-placeholder" : "text-textColor"}`}
+                        role="combobox"
+                      >
+                        <span className="line-clamp-1 text-left">
+                          {field?.value
+                            ? TIMEZONES?.find(
+                                (zone) => zone?.value === field?.value,
+                              )?.label
+                            : "select"}
+                        </span>
+                        <ChevronDown
+                          className={`h-4 w-4 shrink-0 text-placeholder ${timeZonePopover && "rotate-180"} transition-transform duration-200`}
```

Comment:
these classes are repeated except the conditional so use a variable to store the repeating ones so that later on changing is easy